lib/api.ts

```diff
@@ -47,6 +48,7 @@ function makeSharedAPI(channel: Channel, data: ConnectMessage): BaseExtensionSDK
   const currentLocation = data.location || locations.LOCATION_ENTRY_FIELD
 
   return {
+    cmaAdapter: createAdapter(channel),
```

Comment:
We were originally planning to just call this `adapter` - right?

Reply:
Yes, we thought the name `adapter` is a little generic and therefore went for `cmaAdapter` in case there will be more adapters in the future (e.g. GraphQL adapter?).
Also, the name anyway doesn't match with the `cma.js` as it needs to passed in there as `apiAdapter`